action/src/config.ts

```diff
@@ -11,14 +17,16 @@ export const config : Config = {
   },
 };
 
+export const config : Config = getYamlInput('config') as Config || defaultConfig; // Default to default config if not provided
+
 interface Config {
   api: {
```

Comment:
we should rename this to `appServer` as well and then just set `getYamlInput('app-server')`